tests/e2e/lib.sh

```diff
@@ -486,6 +496,26 @@ export_central_basic_auth_creds() {
     ci_export "ROX_ADMIN_PASSWORD" "$ROX_ADMIN_PASSWORD"
 }
 
+export_central_cert() {
+    # Export the internal central TLS certificate for roxctl to access central
+    # through TLS-passthrough router by specifying the TLS server name.
+    ci_export ROX_SERVER_NAME "central.${CENTRAL_NAMESPACE:-stackrox}"
+
+    require_environment "API_ENDPOINT"
+    require_environment "ROX_ADMIN_PASSWORD"
+
+    local central_cert
+    central_cert="$(mktemp -d)/central_cert.pem"
+    info "Storing central certificate in ${central_cert} for ${API_ENDPOINT}"
+    stacktrace
```

Comment:
Why do you print the stacktrace here? I this intentional or left over from debugging?

Reply:
Ah, I liked the `stacktrace` function so much while debugging that didn't realize it was not needed anymore.